components/configurator/ui/ConfiguratorRecaptcha.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components'
+import GoogleReCAPTCHA from 'react-google-recaptcha'
+import { useEffect, useRef } from 'react'
+import ConfiguratorWrapper from './ConfiguratorWrapper'
+import useWeb3 from '../../hooks/useWeb3'
+
+const StyledGoogleReCAPTCHA = styled(GoogleReCAPTCHA)`
+  opacity: 0;
+  visibility: hidden;
```

Comment:
Just wondering if this is better than `display: none` ?

Reply:
It needs to be in the page and visible to screen readers as this is similar to: https://github.com/twos-complement/level-protocol-website/blob/e60dabeb84136be2e1bbc74249b28689ed87d3c1/components/ui/Typography.js#L139-L147